tests/test_xopen.py

```diff
@@ -576,6 +576,18 @@ def test_xopen_zst_fails_when_zstandard_not_available(monkeypatch):
             f.read()
 
 
+@pytest.mark.parametrize("threads", (None, 0, 1))
+def test_xopen_zst_long_window_size(threads):
+    if threads == 0 and zstandard is None:
+        return
+    elif threads == 1 and not shutil.which("zstd"):
+        return
+    test_zst = Path(__file__).parent / "only_zeroes.zst"
+    with xopen(test_zst, "rb", threads=threads) as f:
+        data = f.read()
```

Comment:
This takes 0.5s on my system, which is impressively fast given that 2 GB of data are decompressed, but it’s still quite a long time for a test. I think it would be nicer to just read a couple of bytes from the start of the file. That is enough to detect whether the window size has been set correctly.

Reply:
I only read 1024 bytes now and check if they are zero. This works wonders for the zstandard.open function. Unfortunately the command line application still takes a second to output its first data into stdout. Since there is no other way to test this properly I guess that is acceptable?

Reply:
Fine with me